storybook/stories/components/Icons.stories.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { ComponentType } from 'react'
+import { Icon, IconName, iconsIndex, Monospace, styled, theme } from '@sfgov/react'
+import { ComponentMeta, Story } from '@storybook/react'
+
+type IconArgs = {
+  symbol: IconName
+  color: string
+  width?: number
+  height?: number
+}
+
+const sortedKeys = Object.keys(iconsIndex).sort() as IconName[]
+
+const colorKeys = Object.keys(theme.colors)
+const colorMapping = Object.fromEntries(
+  colorKeys.map(key => [key, `$${key}`])
+)
+
+export default {
+  component: Icon,
+  args: {
+    symbol: 'accessibility',
+    color: 'slateL4',
+    width: 20
+  },
+  argTypes: {
+    symbol: {
+      name: 'Symbol',
+      options: sortedKeys,
```

Comment:
currently this shows up as the index of each entry, would the `iconsByName` export work here instead?